### PR TITLE
Basic winner screen

### PR DESCRIPTION
**Pull request by TitouanUniLu**
very basic winner: pop up window with crown gif, with a text saying which team won (the background color changes depending on the winner).
One issue: in case of draw or "timeout", two pop up windows appear: one for each team since they are both considered winners 
No other bugs found, works with the latest version of the code

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,9 @@
 # Mobile Tools for Java (J2ME)
 .mtj.tmp/
 
+#temporary fix for the startserver
+StartServer.java
```
Comment (ptal) [inline .gitignore:14]: remove this line please

File: src/main/java/lol/server/Server.java [hunk 2]
```diff
@@ -68,6 +68,13 @@ private void endOfGameMessage() {
       Nexus nexus = battlefield.nexusOf(i);
       if(nexus.isAlive()) {
         System.out.println("WINNER: " + nexus);
+        String winner = "WINNER: " + nexus;
```
Comment (ptal) [inline src/main/java/lol/server/Server.java:71]: you could simply write:

```java
if(nexus.teamOfNexus() == Nexus.BLUE) { ... }
else { ... }
```

File: src/main/java/lol/ui/WinnerScreen.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
+package lol.ui;
+import javax.swing.*;
+import java.awt.*;
+
+public class WinnerScreen {
```
Comment (ptal) [inline src/main/java/lol/ui/WinnerScreen.java:5]: This file is using Java Swing which is another Java graphical component.

It is not a big deal, for if you want me to merge your code, you should use Java FX instead.

(You will still get bonus points if you don't change it).

**Review by ptal — CHANGES_REQUESTED** [review]
Cool!

However, you must not add the .idea and target folders to your commit. These files shouldn't be pushed.

File: src/main/java/lol/server/Server.java (hunk 2 above)
Comment (TitouanUniLu) [reply src/main/java/lol/server/Server.java:71]: done!

**Pull request closed without merging**